app/routes/__index/quest/index.tsx

```diff
@@ -0,0 +1,58 @@
+import PlusIconWhite from "~/components/Icons/PlusIcon";
+import { LinkFooter } from "~/components/Buttons/LinkButton";
+import Carousel from "~/components/Quest/Carousel";
+import { db } from "~/utils/db.server";
+import { useLoaderData } from "@remix-run/react";
+import { json } from "@remix-run/node";
+
+
+
+
+export async function loader() {
+    const allProjects = await db.badges.findMany({
```

Comment:
From your comment.
Did you make a prisma db push to have the badges available here?

Reply:
yes no more issue magic